proxy.go

```diff
@@ -95,21 +96,23 @@ type client struct {
 	conn net.Conn
 }
 
-func (c *client) info(lvl glog.Level, msg string) {
-	if !glog.V(lvl) {
-		return
-	}
-	glog.Infof("[client #%d] %s", c.id, msg)
-}
+var nextClientID = uint64(0)
```

Comment:
not quite, I splitted #56 and this PR will need to be rebased on top (as noted in the PR message).